### DISCUSSION TITLE
[1.8] 2022-12-13

### DISCUSSION THREAD

**Discussion by darthcloud** [darthcloud/BlueRetro#490] [General]
## Global
* [MGR] Add support for internal install for all system (Only tested GC)
* [GAMEID] Add support for Game ID for PS1, N64 & GC multi config selection
* [ADAPTER] Reduce 8bit accessible heap memory usage (Fix memcard malloc in universal build)

## Wired
* [GC] Support all command format

## Bluetooth
* [BT] Add support for 8bitdo M30 controller
* [BT] Add RetroFighters GC adapter support
* [ADAPTER] Fixup 8bitdo Gbros B & X mapping

# Firmware is now available in 2 different flavor
## 1. Regular FW as before retroactively renamed HW1 (Typically for external adapter)
Use firmwares from HW1 archive for:
* [WillConsole's](https://twitter.com/WillConsole) PSUnoRetro
* All [Humble Bazooka's](https://twitter.com/humblebazooka) BT dongle
* [RetroTime's](https://twitter.com/bixxewoscht) N64 dongle
* [RetroOnyx's](https://twitter.com/mellott124) VB dongle
* [GrechTech's](https://twitter.com/GrechTech) RetroRosetta
* any DIY adapter you made base on the original BlueRetro spec
* most likely anything from aliexpress

## 2. HW2 FW that support power management and port detection (Typically for internal install)
Use firmwares from HW2 archive for:
* [Laser Bear Industries's](https://twitter.com/collingall) GameCube BlueRetro Internal Adapter
* any DIY internal install you made base on the HW2 spec

## [OTA Firmware update page](https://blueretro.io/ota.html) has protection to prevent cross flashing between HW1 & HW2 firmware.

# Download latest firmware archives from https://darthcloud.itch.io/blueretro

<hr /><em>This discussion was created from the release <a href='https://github.com/darthcloud/BlueRetro/releases/tag/v1.8'>[1.8] 2022-12-13</a>.</em>

**Comment by aquagiraffe** [comment]


**Reply by darthcloud** [reply]
can you take a screenshot of the OTA page?

**Reply by aquagiraffe** [reply]
Apologies, it seems it did update but gave the error message anyway. Disregard!

**Reply by darthcloud** [reply]
You saw the line going all the way to 100%? (takes ~10min)

**Reply by aquagiraffe** [reply]
No, didn't get a progress bar and it definitely didn't go 10 min (was maybe seconds) but when connecting to it, the OTA page now lists it as being on 1.8 RC4

**Reply by darthcloud** [reply]
You need to unzip the archive!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Reply by aquagiraffe** [reply]
Whoops!
